ironfish/src/blockchain/blockchain.test.ts

```diff
@@ -732,17 +732,6 @@ describe('Blockchain', () => {
     })
   })
 
-  it('reject block with null previous hash', async () => {
```

Comment:
Removing this test because `verifyBlockAdd` can no longer be called without a previous block